### PR TITLE
Adds Advisor Recommendations card

### PR DESCRIPTION
**Pull request by AllenBW**
s all done, good enough for a first cut, **everything is live**, n should redirect like a 💎 

fixes https://projects.engineering.redhat.com/browse/RHCLOUD-4638

~despite what mocks say, gonna rename this card `Advisor Recommendations` as per BU request
follows mocks here https://marvelapp.com/a5c8bcg/screen/66136610~

~TODO~
* ~figure out what the chart theme is (colors), implement it in both legend and chart~
* ~bullet chart order is dynamic, smallest to largest, figure out if this is ok or if it needs to be static~
* ~hook up the api, easy peasy 🍋 squeezy~
* ~make chart clicks and legend clicks to go insights rules table with correct filter set~
* solve the Pin nut Crisis 2020

#### omg its live and amazing
Screen Shot 2020-03-11 at 12 21 18 PM
Screen Shot 2020-03-11 at 12 17 20 PM

### CODE REVIEW

**Comment by AllenBW** [comment]
cc @kybaker

File: src/SmartComponents/Advisor/Advisor.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    advisorIncidents, advisorIncidentsStatus, advisorFetchIncidents, intl }) => {
+
+    useEffect(() => {
+        advisorFetchStatsRecs();
+        advisorFetchStatsSystems();
+        advisorFetchIncidents();
+    }, [advisorFetchIncidents, advisorFetchStatsRecs, advisorFetchStatsSystems]);
+
+    return <TemplateCard appName='Advisor'>
+        <TemplateCardHeader title='Advisor Recommendations'>
```
[the remark this answers is not in the document]
Comment (AllenBW) [reply src/SmartComponents/Advisor/Advisor.js]: @ryelo good question for @kybaker

[the remark this answers is not in the document]
Comment (kybaker) [reply src/SmartComponents/Advisor/Advisor.js]: @ryelo Yes sentence case here

**Comment by christiemolloy** [comment]
Is this needed anymore I dont see it in the mocks

Screen Shot 2020-03-11 at 1 27 15 PM

**Comment by AllenBW** [comment]
@christiemolloy which mocks yah looking at? I see it in https://marvelapp.com/a5c8bcg/screen/66136610

File: build/messages/src/Messages.json [hunk 2]
```diff
@@ -3,5 +3,35 @@
     "id": "dashboardTitle",
     "description": "Title of the dashboard",
     "defaultMessage": "Overview"
+  },
+  {
+    "id": "recsImpactingSystems",
+    "description": "Advisor - recs impacting systems",
+    "defaultMessage": "{totalRecs} recommendations impacting {systems, plural, one {# system} other {# systems}}"
```
Comment (ryelo) [inline build/messages/src/Messages.json]: `recommendations` needs to change to `recommendation` if it's 1 totalRecs

File: locales/data.json [hunk 3]
```diff
@@ -1,5 +1,11 @@
 {
   "en": {
-    "dashboardTitle": "Overview"
+    "dashboardTitle": "Overview",
+    "recsImpactingSystems": "{totalRecs} recommendations impacting {systems, plural, one {# system} other {# systems}}",
```
Comment (ryelo) [inline locales/data.json]: `recommendations` needs to change to `recommendation` if it's 1 totalRecs

File: locales/translations.json [hunk 4]
```diff
@@ -1,3 +1,9 @@
 {
-  "dashboardTitle": "Overview"
+  "dashboardTitle": "Overview",
+  "recsImpactingSystems": "{totalRecs} recommendations impacting {systems, plural, one {# system} other {# systems}}",
```
Comment (ryelo) [inline locales/translations.json]: `recommendations` needs to change to `recommendation` if it's 1 totalRecs

File: src/Messages.js [hunk 5]
```diff
@@ -5,5 +5,35 @@ export default defineMessages({
         id: 'dashboardTitle',
         description: 'Title of the dashboard',
         defaultMessage: 'Overview'
+    },
+    recsImpactingSystems: {
+        id: 'recsImpactingSystems',
+        description: 'Advisor - recs impacting systems',
+        defaultMessage: '{totalRecs} recommendations impacting {systems, plural, one {# system} other {# systems}}'
```
Comment (ryelo) [inline src/Messages.js]: `recommendations` needs to change to `recommendation` if it's 1 totalRecs

**Comment by christiemolloy** [comment]
I realized this morning those mocks were archived, the latest mocks are: https://marvelapp.com/a5c8bcg/screen/66441550 @AllenBW , right @kybaker

**Comment by kybaker** [comment]
@christiemolloy Yes that is the latest. I am working on updating the mock again with the correct text. I will send the latest when I have it.

**Comment by AllenBW** [comment]
yeesh, surprise new mocks, y'all are too much fun for me, ok will update

**Comment by AllenBW** [comment]
new ss to match new mock, gonna have to work on getting the text in the legend of the chart @kybaker so it won't be in this pr

Screen Shot 2020-03-11 at 4 54 56 PM

**Pull request merged**